yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/call/Receipt.java

```diff
@@ -95,12 +95,12 @@ public Receipt build() {
         }
     }
 
-    public static enum Outcome {
+    public enum Outcome {
```

Comment:
didn't we drop all the enums?

Reply:
Correct.  We did.  Well spotted.
The idea is that we will not use Enums for attribute values - we will not return Enum's to SDK clients.
This one here is used internally.  Values from the Receipt will be mapped to some implementation of _Profile_, which will not use Enums.  I think this should be ok.  There are similar uses of Enums in the xxxProto classes, which we have no control over.